Samples/Use legacy custom panel in 7.4/Resources/SamplePanelDefinition.cs

```diff
@@ -0,0 +1,27 @@
+using Litium.Foundation.Modules.ECommerce;
+using Litium.Web.Administration.Panels;
+
+namespace Litium.Accelerator.Mvc.Site.ECommerce.Panels
+{
+    /// <summary>
+    /// Defines panel definition, to display it in EcommerceArea.
+    /// </summary>
+    public class SamplePanelDefinition : PanelDefinitionBase<EcommerceArea, SamplePanelDefinition.SettingsModel>
+    {
+        /// <summary>
+        /// Points to the path of SamplePanelPage.aspx
+        /// </summary>
+        public override string Url => "https://localhost/Site/ECommerce/Panels/SamplePanelPage.aspx";
```

Comment:
Could this property be explained? I do not understand what url to set here and why?

Reply:
Maybe use relative url?

Reply:
People should have basic knowledge of how to define a panel using panel definition by reading this first https://docs.litium.com/documentation/architecture/back-office_1/creating-custom-panel

Anyway, I added the link to the field's summary.